web-client/public/js/setup-handlers.js

```diff
@@ -8,6 +8,9 @@ import {
     WEIGHTS_SHOW_MOUSE_OVER_CLASS,
     WEIGHTS_SHOW_ALWAYS_CLASS,
     WEIGHTS_HIDE_CLASS,
+    showWeightsMouseover,
+    showAllWeights,
+    hideAllWeights,
```

Comment:
So, notice how these new constants kind of stick out like a sore thumb when compared to the constants that came before them 🤔